UPGRADING.rst

```diff
@@ -10,6 +10,13 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#2025 Register indexer service accounts with SAM
+================================================
+
+Service accounts are now registered with SAM upon deployment.
+They can also manually be registered via ``make -C terraform sam``.
+However, the accounts must still be manually authorized for steward access by
+the team lead.
```

Comment:
This should go in the README since the need to get authorized does not only occur during upgrades but for every new deployment.